README.md

```diff
@@ -42,15 +42,15 @@ To get a local copy up and running follow these simple example steps.
 
```

Comment:
- [ ] [OPTIONAL] Kindly make sure to add some kind of instruction on how to use the app, and consider removing all the unused headers from your README file so it looks more professional
![Screenshot_20221110_211129](https://user-images.githubusercontent.com/25227022/201195913-f8cb2e6f-ac88-41b1-bd6f-703d245943ca.png)